sdk/monitor/azquery/MIGRATION.md

```diff
@@ -0,0 +1,180 @@
+# Guide to migrate from `operationalinsights` and monitor `insights` to `azquery`
+
+This guide is intended to assist in the migration to the `azquery` module. `azquery` allows users to retrieve log and metric data from Azure Monitor.
+
+## Package consolidation
+
+ Azure Monitor allows users to retrieve telemetry data for their Azure resources. The main two data catagories for Azure Monitor are [metrics](https://learn.microsoft.com/azure/azure-monitor/essentials/data-platform-metrics) and [logs](https://learn.microsoft.com/azure/azure-monitor/logs/data-platform-logs). 
+ 
+ There have been a number of [terminology](https://learn.microsoft.com/azure/azure-monitor/terminology) changes for Azure Monitor over the years which resulted in the operations being spread over multiple packages. For Go, metrics methods were contained in `github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/<version-number>/insights` and logs methods resided in `github.com/Azure/azure-sdk-for-go/services/operationalinsights/v1/operationalinsights`.
+
+The new `azquery` module condenses metrics and logs functionality into one package for simpler access. The `azquery` module contains two clients: LogsClient and MetricsClient.
+
+Transitioning to a single package has resulted in a number of name changes, as detailed below.
+
+### Log name changes
+
+| `operationalinsights`    | `azquery` |
+| ----------- | ----------- |
+| QueryClient.Execute      | LogsClient.QueryWorkspace     |
+| MetadataClient.Get and MetadataClient.Post | N/A |
+
+The `azquery` module does not contain the `MetadataClient`. For that functionality, please use the old `operationalinsights` module or [file an issue in our github repo](https://github.com/Azure/azure-sdk-for-go/issues), so we can prioritize adding it to `azquery`.
```

Comment:
Probably worth providing the pkg.go.dev link to the operationalinsights package.

Reply:
fixed in 36b33f8